CONTRIBUTING.md

```diff
@@ -57,8 +57,8 @@ See ESLint's [RuleTester](https://eslint.org/docs/developer-guide/nodejs-api#rul
 for more on the allowable properties (e.g., `code`, `errors` (for invalid rules),
 `options`, `settings`, etc.).
 
-Note that besides `npm test`, there is `npm run test-cov` which shows more
-detailed information on coverage. Coverage should be maintained at 100%, and
+Note that besides `npm test` there is `npm run test-cov`, which shows more
```

Comment:
```suggestion
Note that besides `npm test` there is `npm run test-cov` which shows more
```

I don't think there is need for a comma here.

Reply:
Thrown the comma in front of "which" because it is introducing a non-defining/non-restrictive clause.

I think the following article explain this change far better than I can!:

- [Grammarly - Comma Before Which](https://www.grammarly.com/blog/comma-before-which/ )